### ISSUE TITLE
Accessing exported modules from require path

### ISSUE THREAD

**Issue by timkinnane** [zkochan/package-preview#31]
I'm trying to use preview to test a submodule, as per this article:
https://jaketrent.com/post/import-parts-node-packages-slash/

Should this be possible? Can't tell if its a limitation of preview-package or I'm not building the package properly.

For context, I'm trying to make a module in my package available by requiring its path, e.g.

Instead of: `require('myApp').start()`

I would like `require('myApp/start')`

But I'm getting the error `Cannot find module 'myApp/start'`

I think I've achieved this by adding a start.js file that invokes the start method as: `module.exports = app.start()` and I can execute it from command line via `node src/start`.

**Comment by zkochan** [comment]
Yes, this should be possible if `start.js` is published with your package (it is not ignored by `.npmignore`).

If it is published, please create a simple repo to reproduce the issue and I'll check
